adserver/templates/adserver/email/flight-renewing.html

```diff
@@ -17,6 +17,14 @@
   {% endblocktrans %}
 </p>
 
+{% if flight.auto_renew_payment_method %}
+<p>
+  {% blocktrans with payment_method=flight.auto_renew_payment_method|lower %}
+  For your new flight we will {{ payment_method }}.
```

Comment:
This is likely going to be have caps -- I'd probably just make this more formal?

```suggestion
  Payment method: {{ payment_method }}.
```